src/main/java/com/orange/main/BaseController.java

```diff
@@ -0,0 +1,17 @@
+package com.orange.main;
+
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BaseController {
+/* Transfer Object to ReponseObj, make Object to JSON correctly. */
+    protected static ResponseObj transResponseObj(Object obj){
+        ResponseObj rtnObj = new ResponseObj();
+        rtnObj.setCode("200");
+        rtnObj.setData(obj);
+
+        return rtnObj;
+    }
+
+    
```

Comment:
It seems you lack the error format model, like below

``` JSON
{
   "code":  ...,
   "extra":  ...,  // optional
   "message":  ...,
}
```

Reply:
Spring Boots responds JSON format is following attributes from Object.
You can see "ReturnObj".

Reply:
I can't get it, would you tell me more details?